xml/Microsoft.Build.Tasks.Xaml/CompilationPass2Task.xml

```diff
@@ -185,7 +185,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets or sets whether the task in in process of compiling XAML markup.</summary>
-        <value>True if the task in in process of compiling XAML markup.; otherwise, false.</value>
+        <value><see langword="true" /> if the task in in the process of compiling XAML markup; otherwise, <see langword="false" />.</value>
```

Comment:
```suggestion
        <value><see langword="true" /> if the task is in the process of compiling XAML markup; otherwise, <see langword="false" />.</value>
```